sdk/websites/Azure.ResourceManager.AppService/src/autorest.md

```diff
@@ -849,4 +849,40 @@ directive:
             "Aborted",
             "Running"
         ]
+  - from: WebApps.json
+    where: $.paths['/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/hybridConnectionRelays'].get
+    transform: >
+        $['operationId'] = "WebApps_GetAllTheHybridConnectionData";
+        $['responses'] = {
+          "200": {
+            "description": "Description for Retrieves all Service Bus Hybrid Connections used by this Web App.",
+            "schema": {
+              "$ref": "./AppServicePlans.json#/definitions/HybridConnectionCollection"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        };
+  - from: WebApps.json
+    where: $.paths['/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slot}/hybridConnectionRelays'].get
+    transform: >
+        $['operationId'] = "WebApps_GetAllTheHybridConnectionSlotData";
+        $['responses'] = {
```

Comment:
As @archerzz  mentioned, just fix the property that have wrong value, here is
 $['responses']['200']['schema']['$ref'] = "./AppServicePlans.json#/definitions/HybridConnectionCollection";